met-web/src/services/widgetService/DocumentService.tsx/index.tsx

```diff
@@ -32,3 +32,51 @@ export const postDocument = async (widget_id: number, data: PostDocumentRequest)
         return Promise.reject(err);
     }
 };
+
+export const deleteDocument = async (widget_id: number, document_id: number): Promise<DocumentItem> => {
+    try {
+        const url = replaceAllInURL({
+            URL: Endpoints.Documents.DELETE,
+            params: {
+                document_id: String(document_id),
+                widget_id: String(widget_id),
+            },
+        });
+        const response = await http.DeleteRequest<DocumentItem>(url);
+        if (response.data.status && response.data.result) {
```

Comment:
you could probably get away without this if statement. Perhaps lookup all the other delete service methods and maybe check if all of them have it

Reply:
it seems it's the same in all of them